setup/cmd/root.go

```diff
@@ -70,6 +71,7 @@ func Execute() {
 	cmd.Flags().BoolVar(&skipCSVGen, "skip-csvgen", false, "if an all-namespaces operator should be installed to generate a CSV resource in each namespace")
 	cmd.Flags().BoolVar(&skipDefaultTemplate, "skip-default-template", false, "skip the setup/resources/user-workloads.yaml template file when creating resources")
 	cmd.Flags().IntVar(&operatorsLimit, "operators-limit", len(operators.Templates), "can be specified to limit the number of additional operators to install (by default all operators are installed to simulate cluster load in production)")
+	cmd.Flags().StringVarP(&idlerTimeout, "idler-timeout", "i", "12h", "overrides the default idler timeout")
```

Comment:
Let's maybe keep it 15s by default and instruct the serverless folks how to use this param to set it to 12h?

Reply:
Done https://github.com/codeready-toolchain/toolchain-e2e/pull/376/commits/21e5abd0c1f02f91f6961c32808a39bb48ef2f12